web/src/sdk/mutations/QueueMutationLink.js

```diff
@@ -57,7 +58,22 @@ export class QueueMutationLink extends ApolloLink {
     //store only if there are values for query.definitions
     if (definitions.length > 0) {
       query.definitions = definitions
-      this.queue.push({mutation: query, variables})
+      let operationName = query.definitions[0].name.value
```

Comment:
Not guaranteed to be always there. Let's add some if's to check if things are not undefined.